policyengine_us/variables/gov/states/ca/cdss/tanf/cash/eligibility/ca_tanf_financial_eligible.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class ca_tanf_financial_eligible(Variable):
+    value_type = bool
+    entity = SPMUnit
+    label = "California CalWORKs Financial Eligibility"
+    definition_period = YEAR
+    defined_for = StateCode.CA
+
+    def formula(spm_unit, period, parameters):
+
+        tanf_enrolled = spm_unit("is_tanf_enrolled", period)
+        applicant_test = spm_unit("ca_tanf_applicant_financial_test", period)
+        recipient_test = spm_unit("ca_tanf_recipient_financial_test", period)
+
+        return where(
```

Comment:
are you sure applicants also have to pass the recipient test? i thought the recipient test was really just that `calculated_tanf > 0`, plus that they continue to pass other parts of the test like assets.

if possible to unify this so that the income limits are *based on recipient status* but don't need separate variables, it would simplify the code considerably

Reply:
"If the remaining income is less than the MBSAC, then the family moves onto the Recipient EID Test where the recipient disregards will be applied." Eligible applicants may move forward to the next test, which is actually checking whether countable income < maximum payment. Do you think it is better to disregard the 2nd financial test and think it as a way to compute the final benefit?

Reply:
I think the question is whether the last sentence of the applicant eligibility implies that the applicant has to pass the recipient test or if it is a foreshadowing of the following paragraph about the recipient test 
![Uploading image.png…]()

Reply:
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/65aa2236-773f-42da-9b50-313f2b3a0606)